content/en/community/_index.html

```diff
@@ -121,7 +121,7 @@ <h1 style="padding-top:1%">Discussions</h1>
 <div class="resourcecontainer">
 
 <div class="resourcebox">
-<img src="/images/community/discuss.png" style="width:80%;padding-bottom:2%">
+<img src="/images/community/discuss.png" alt="Discuss" style="width:80%;padding-bottom:2%">

```

Comment:
This should read `Forum` as per the guidelines

Reply:
@Rajakavitha1  I'm going to add ``Discuss Forum``. Let me know your thoughts.

Reply:
@ChaitanyaKaranam These guidelines are written by accessibility experts so let us stick to what the experts guide us " https://www.w3.org/WAI/tutorials/images/textual/" not by my thoughts :)

Reply:
Makes sense. Included this in the latest commit.